CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# 0.7.0-M2
+
+## Additions
+
+* Re-added `values()` in `Intent` to replace enum values 
+* Re-added `intents` in `Intents` to get of `Intents`
```

Comment:
https://en.wikipedia.org/wiki/Semantic_satiation

Reply:
Ruined cause of Kotlin markdowns I'd say :P